### DISCUSSION TITLE
Update "Programming Language" classifiers

### DISCUSSION THREAD

**Discussion by kasium** [pradyunsg/furo#625] [Ideas] [answered]
The "Programming Language" seems to be outdated. Python 3.5/3.6 are not supported anymore and 3.10/3.11 are but are not part of the list. Can you please update it?

**Comment by dgw** [comment, marked as answer]
In hopes that it would make @pradyunsg's life a bit easier, I went ahead and opened #626 so this idea can be accepted or rejected "with one click" by merging or closing the PR. 🙂

**Reply by pradyunsg** [reply]
Thank you for this!

**Reply by kasium** [reply]
Thanks for the fast support
